linera-chain/src/chain.rs

```diff
@@ -556,6 +556,7 @@ where
     where
         F: Fn(E) -> Message,
     {
+        use futures::stream::{self, StreamExt, TryStreamExt};
```

Comment:
Why not include it globally, with the other imports?

Reply:
Indeed.